spec/models/game_spec.rb

```diff
@@ -0,0 +1,65 @@
+require 'rails_helper'
+
+RSpec.describe Game, type: :model do
+  it 'has 32 pieces' do
+    game = Game.create(id: 1, white_player_id: 1, black_player_id: 2)
+    piece_count = Piece.count
+    game_count = Game.count
+    #Game.last.pieces.each {|piece| p piece}
+    expect(game_count).to eq 1
+    expect(piece_count).to eq 32
+  end
+
+  it 'has 16 white pieces & 16 black pieces' do
+    game = Game.create(id: 1, white_player_id: 1, black_player_id: 2)
+    game_count = Game.count
+    white_piece_count = Piece.where(:color => 'white').count
+    black_piece_count = Piece.where(:color => 'black').count
+    expect(game_count).to eq 1
+    expect(white_piece_count).to eq 16
+    expect(black_piece_count).to eq 16
+  end
+
+
+  describe "game pieces method is_Obstructed?" do

```

Comment:
The test look good and should be updated to match the logic mentioned above for the is_obstructed method.
The test related to the piece model should be created in a separate spec file --> "piece_spec.rb"